src/ethereum_test/types.py

```diff
@@ -13,7 +13,33 @@
 from .common import AddrAA, TestPrivateKey
 
 
-def hex_or_none(input: Union[int, None], default=None) -> Union[str, None]:
+class UNSPECIFIED(object):
+    """
+    Class used as a default placeholder to detect if a parameter or value was
+    or not provided by the test writer.
+    As opposed to using `None` where the test spec writer wishes to verify that
+    a value is indeed `None`.
+    """
+
+    def __new__(cls):
+        """Create the singleton UNSPECIFIED object"""
+        if not hasattr(cls, "instance"):
+            cls.instance = super(UNSPECIFIED, cls).__new__(cls)
+        return cls.instance
+
+
+def hex_or_none_unspecified(
```

Comment:
Couldn't we just expand `hex_or_none` with an "unspecified" default?

Reply:
Indeed, check out the proposed solution 😉 (spoiler alert: it uses `UNSPECIFIED`)